engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/job/EngineJobExecutionActor.scala

```diff
@@ -623,15 +620,23 @@ class EngineJobExecutionActor(replyTo: ActorRef,
     runJob(updatedData)
   }
 
-  private def requestRestartCheckToken(): Unit =
+  private def requestRestartCheckToken(): Unit = {
+    jobLogger.info(
+      s"EJEA ${self.toString().split('#').lastOption.getOrElse("ERR").stripSuffix("]")} is requesting restart token of type ${backendLifecycleActorFactory.jobExecutionTokenType} for ${workflowDescriptor.backendDescriptor.hogGroup}"
```

Comment:
All this gross string munging turns `Actor[akka://cromwell-system/user/cromwell-service/WorkflowManagerActor/WorkflowActor-f1f2a6a0-9312-4095-8090-466e47f86a65/WorkflowExecutionActor-f1f2a6a0-9312-4095-8090-466e47f86a65/f1f2a6a0-9312-4095-8090-466e47f86a65-EngineJobExecutionActor-test.hello:NA:1#15931988]` into the unique identifier `15931988`. The other relevant info in the actor ref string is already available in the job logger prefix.